src/ssl.c

```diff
@@ -23000,10 +23002,22 @@ int wolfSSL_get_shutdown(const WOLFSSL* ssl)
     WOLFSSL_ENTER("wolfSSL_get_shutdown");
 
     if (ssl) {
-        /* in OpenSSL, WOLFSSL_SENT_SHUTDOWN = 1, when closeNotifySent   *
-         * WOLFSSL_RECEIVED_SHUTDOWN = 2, from close notify or fatal err */
-        isShutdown = ((ssl->options.closeNotify||ssl->options.connReset) << 1)
-                    | (ssl->options.sentNotify);
+#if defined(OPENSSL_EXTRA) || defined(WOLFSSL_WPAS_SMALL)
+        if (ssl->options.handShakeState == NULL_STATE) {
+            /* The SSL object was possibly cleared with wolfSSL_clear after
+             * a successful shutdown. Simulate a response for a full
+             * bidirectional shutdown. */
+            isShutdown = 3;
```

Comment:
What is `3`? Do we have a macro or enum we can use?

Reply:
Changing to use the defined enum values.